server/src/repositories/Training/questionRepo.ts

```diff
@@ -1,67 +1,54 @@
 import { knex } from "../../db";
 import { Question } from "../../models/training/question";
 import { QuestionMap } from "../../mappers/training/questionMapper";
-import { query } from "express";
 
-export interface IQuestionRepo {
-  getQuestionById(questionId: number | string): Promise<Question>;
-  getQuestionsByModule(moduleId: number | string): Promise<Question[]>;
-  addQuestionToModule(
-    moduleId: number | string,
-    question: Question
-  ): Promise<Question>;
-  deleteQuestionById(questionId: number | string): Promise<void>
+export async function getQuestion(
```

Comment:
@pjw9963 I refactored the question and module repos to be a series of exported functions, instead of classes. IMO we weren't getting any benefit from having them as classes, and it was just adding boilerplate. LMK what you think of this instead.

Reply:
Yeah this is fine. I had them as classes for testing and handling transactions but we are not doing either of those so 👍